From 49814267ff57860f8c639e97a37e0fe0b77f0e13 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Wed, 12 Apr 2023 11:51:19 +0800
Subject: [PATCH] u-boot-s32: Aptiv: set PFENG_MODE to be disable by
default

When PFE is configured to work in slave mode in linux, it shouldn't be
enabled in U-Boot because this enablement will trigger master mode
initialization which should be done by M7 Autosar image.

Upstream-Status: Inappropriate [WR Linux specific change for Aptiv autosar]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 include/configs/s32g2xxaevb.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32g2xxaevb.h b/include/configs/s32g2xxaevb.h
index 499cf7e6d0..2e9c3f9ca0 100644
--- a/include/configs/s32g2xxaevb.h
+++ b/include/configs/s32g2xxaevb.h
@@ -14,7 +14,7 @@
 
 #define FDT_FILE			"s32g274a-aptiv.dtb"
 #ifdef CONFIG_FSL_PFENG
-#  define PFENG_MODE "enable,sgmii,none,sgmii"
+#  define PFENG_MODE "disable,sgmii,none,sgmii"
 #  define PFENG_EMAC "0"
 #endif
 
-- 
2.40.0

